docs/roadmap/Roadmap.md

```diff
@@ -1,79 +1,75 @@
 
-We're planning on collaborating with AI frameworks to support a new era of AI powered Flutter apps.
+## Core framework
 
-We are not planning on expanding the set of [flutter.dev plugins](https://pub.dev/publishers/flutter.dev/packages) we maintain, but will rather focus on raising the quality of the existing plugins, and resolving core feature gaps (for example, investigating an updated [shared_preferences](https://pub.dev/packages/shared_preferences) API that better supports use of isolates and to add-to-app use case). We'll also support community initiatives like [Flutter Favorites](https://pub.dev/packages?q=is%3Aflutter-favorite).
-
-We'll also continue to add support for building casual games with Flutter, as a joint effort with the [Flame](https://flame-engine.org/) community.
+We're investigating a number of changes with a goal of reducing unnecessary verbosity in Flutter widget code.
 
 ## Tooling and AI
+We'll continue integrate with AI solutions to offer AI assistance for core programming tasks.
 
-We hope to integrate with AI solutions to offer AI assistance for core programming tasks.
-
-We'll also continue to collaborate with Google's [IDX team](https://developers.google.com/idx), and explore integration with design tools.
+We'll continue to invest in our suite of tooling including Flutter DevTools, VS Code, Android Studio/IntelliJ, as well as IDX, with an eye toward always improving the edit-refresh cycle and the overall developer experience.
 
-## Programming language
+## Dart programming language
+In 2024 we concluded that supporting macros in Dart was not viable. Based on that, in 2025 we expect to improve the current support for code generation in build_runner, and to investigate alternatives ways of improving Dart support for serialization and deserialization.
 
-The Dart team expects to complete the assessment of the viability of supporting [macros](https://github.com/dart-lang/language/issues/1482) in Dart, and in 2024 either ship the first phases of supporting them, or if we discover unmitigable architectural issues, abandon the effort. Key use cases for macros include serialization/deserialization, data classes, and general extensibility.
+We also expect to ship one or more language features currently going through the Dart language design funnel.
 
-We'll investigate a number of more incremental language features, such as syntax changes to reduce verbosity (for example, [primary constructors](https://github.com/dart-lang/language/issues/2364) and [import syntax shorthand](https://github.com/dart-lang/language/issues/649)), and better support for statically checked variance.
+## Dart compilers and tools
+We'll work on refactoring the Dart analyzer and the front-end compiler to share more of their implementations, with the goal of accellerating future language feature development, and to make them more performant and stable.
```

Comment:
```suggestion
We plan to refactor the Dart analyzer and the front-end compiler to share more of their implementations so that this can accelerate future language feature development, performance, and stability.
```